proxy/prefect_flows.py

```
@@ -340,39 +340,41 @@ def shellopjob(task_config: dict, task_slug: str):  # pylint: disable=unused-arg
 #     }
 # }
 @flow
-def deployment_schedule_flow_v4(config: dict, dbt_blocks: list = [], airbyte_blocks: list = []):
+async def deployment_schedule_flow_v4(
+    config: dict, dbt_blocks: list = [], airbyte_blocks: list = []
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)